Gemfile

```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
 
+gem "rspec"
+gem "pronto"
 gem "querly"
+gem "pronto-querly"
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pronto-querly` should appear before `querly`.